### PR TITLE
dist/tools/commit-msg: annotate errors in Github Action

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
Another one of those PRs adding annotations for a check.

### Testing procedure
The bogus commits attached should generate annotations

### Issues/PRs references
None

### CODE REVIEW

File: dist/tools/commit-msg/check.sh [hunk 1]
```diff
@@ -6,9 +6,34 @@
...
 # directory for more details.
 
+. "$(dirname "$0")/../ci/github_annotate.sh"
+
+github_annotate_setup
+
 MSG_MAX_LENGTH=50
 MSG_STRETCH_LENGTH=72
 
+if tput colors &> /dev/null && [ $(tput colors) -ge 8 ]; then
```
Comment (MrKevinWeiss) [inline dist/tools/commit-msg/check.sh:16]: It seems something in here makes me print 256 first...

**Review by MrKevinWeiss — COMMENTED** [review]
Running locally I get some number printed?
This PR
<details>

```
./dist/tools/commit-msg/check.sh 
256
Warning: Commit message is longer than 50 (but < 72) characters:
    "fixup! dist/tools/commit-msg: annotate errors in Github Action"
Warning: Commit message is longer than 50 (but < 72) characters:
    "fixup! dist/tools/commit-msg: annotate errors in Github Action"
Warning: Commit message is longer than 50 (but < 72) characters:
    "fixup! dist/tools/commit-msg: annotate errors in Github Action"
Warning: Commit message is longer than 50 (but < 72) characters:
    "fixup! dist/tools/commit-msg: annotate errors in Github Action"
Warning: Commit message is longer than 50 (but < 72) characters:
    "fixup! dist/tools/commit-msg: annotate errors in Github Action"
Warning: Commit message is longer than 50 (but < 72) characters:
    "REMOVE ME! This is just a test to check if a warning is triggered"
Error: Commit message is longer than 72 characters:
    "REMOVE ME! The answer to life, the universe, and everything is the number fourty-two"
Warning: Commit message is longer than 50 (but < 72) characters:
    "dist/tools/commit-msg: annotate errors in Github Action"
Warning: Commit message is longer than 50 (but < 72) characters:

```
</details>


vs master

<details>

```
./dist/tools/commit-msg/check.sh 
Error: Commit message is longer than 72 characters:
    "REMOVE ME test test test test test testasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdf"
Warning: Commit message is longer than 50 (but < 72) characters:
    "lostandfound: document removal of boards/slwstk6220a"
Warning: Commit message is longer than 50 (but < 72) characters:
    "sock/dtls: add example of a blocking handshake to the doc"
Warning: Commit message is longer than 50 (but < 72) characters:
    "pkg/tinydtls: implement sock_dtls_session_set_udp_ep"
Warning: Commit message is longer than 50 (but < 72) characters:

```
</details>


Colors are nice!

File: dist/tools/commit-msg/check.sh (hunk 1 above)
Comment (miri64) [reply dist/tools/commit-msg/check.sh]: Fixed. Also colors for POSIX `echo`.

**Review by MrKevinWeiss — APPROVED** [review]
Fixed the number printing, thanks.  Everything looks good.  Please squash and ... you know what to do.

ACK!

**Comment by miri64** [comment]
Squashed and removed test commits.

**Pull request merged**